ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/authorization/auth/AuthServiceJWT.scala

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.server.api.authorization.auth
+
+import java.util.concurrent.{CompletableFuture, CompletionStage}
+
+import com.daml.ledger.participant.state.v1.{AuthService, Claim, ClaimActAsParty, ClaimAdmin, ClaimPublic, Claims}
+import com.digitalasset.daml.lf.data.Ref
+import io.grpc.Metadata
+import org.slf4j.{Logger, LoggerFactory}
+import spray.json._
+
+import scala.collection.mutable.ListBuffer
+import scala.util.Try
+
+/** The JWT token payload used in [[AuthServiceJWT]]
+  *
+  * TODO(RA): Do we need to wrap everything in a "com.daml.ledger.api" field,
+  * to prevent name collisions with other JWT fields?
+  *
+  * For forward/backward compatibility reasons, all fields are optional.
+  */
+case class AuthServiceJWTPayload(
+  ledgerId: Option[String],
+  applicationId: Option[String],
+  admin: Option[Boolean],
+  actAsParty: Option[List[String]]
```

Comment:
@da-tanabe we can infer `party` from the `actAsParty` iff the List contains one element, else it will need to be provided either in the JSON command/payload or in HTTP header. Same thing with `ledgerId` and `applicationId`.